padertorch/contrib/examples/source_separation/pit/data.py

```diff
@@ -35,49 +42,57 @@ def prepare_dataset(
     return dataset
 
 
-def read_audio(example, src_key="audio_path", audio_keys=None):
+
+def read_audio(example):
+    """
+    Loading of the audio data for an element of the dataset.
+
+    Args:
+        example: for which the audio is loaded
+
+    Returns:
+        example with loaded audio data
+    """
+
+    audio_keys = ['observation', 'speech_source']
+
     data = {
         audio_key: pb.io.audioread.recursive_load_audio(
-            example[src_key][audio_key],
+            example['audio_path'][audio_key],
         )
         for audio_key in audio_keys
     }
-    example["audio_data"] = data
+    example['audio_data'] = data
     return example
 
 
-def pre_batch_transform(inputs, return_keys=None):
+def pre_batch_transform(inputs):
+    """
```

Comment:
Could you also explain our nomenclature in the doc string?
s: speech source
x: speech image
y: observation
Capital letters: STFT of signal